packages/react/src/components/DatePickerInput/DatePickerInput.js

```diff
@@ -30,13 +30,24 @@ export default class DatePickerInput extends Component {
      * control
      */
     labelText: PropTypes.node.isRequired,
+
+    /**
+     * Provide text to be read to screen readers so we can tell user that
+     * he/she can start using calendar dropdown with down arrow key
+     */
+    calendarDropdownAssistiveText: PropTypes.string,
   };
 
   static defaultProps = {
     pattern: '\\d{1,2}\\/\\d{1,2}\\/\\d{4}',
     type: 'text',
     disabled: false,
     invalid: false,
+    calendarDropdownAssistiveText: [
+      'Use down arrow key to navigate to calendar dropdown.',
```

Comment:
Does it make sense for us to specify arrow keys as the interaction mechanism when someone may be using VO on iOS?